pkg/driver/node/credentialprovider/provider_driver.go

```diff
@@ -45,10 +46,11 @@ func (c *Provider) provideFromDriver(provideCtx ProvideContext) (envprovider.Env
 		}
 	}
 
-	// STS Web Identity provider
+	// STS Web Identity provider (IRSA)
 	webIdentityTokenFile := os.Getenv(envprovider.EnvWebIdentityTokenFile)
 	roleARN := os.Getenv(envprovider.EnvRoleARN)
 	if webIdentityTokenFile != "" && roleARN != "" {
+		klog.Info("STS Web Identity provider (IRSA)")
```

Comment:
We should also add log lines to other credentials as well, and we should also mention these are driver-level in the logs